Makefile

```diff
@@ -35,7 +35,8 @@ else
 endif
 
 GO_BUILD_OPTIONS := --tags "netgo osusergo"  -ldflags "-s -X $(NMI_VERSION_VAR)=$(NMI_VERSION) -X $(MIC_VERSION_VAR)=$(MIC_VERSION) -X $(GIT_VAR)=$(GIT_HASH) -X $(BUILD_DATE_VAR)=$(BUILD_DATE) -extldflags '-static'"
-E2E_TEST_OPTIONS := -count=1 -v -timeout 24h -ginkgo.failFast
+# TODO (@aramase) remove the skip test once https://github.com/Azure/aad-pod-identity/issues/214 is resolved
+E2E_TEST_OPTIONS := -count=1 -v -timeout 24h -ginkgo.failFast -ginkgo.skip="should not alter the system assigned identity after creating and deleting pod identity"
```

Comment:
Adding this so others don't hit a known failure while running e2e. I'll remove it once we fix issue #214

Reply:
Thank you @aramase